Documentation/flexvolume.md

```diff
@@ -81,6 +81,11 @@ If you want to use Rook with an already provisioned Tectonic cluster, please ref
 
 Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
 
+### OpenStack Magnum
+OpenStack Magnum using Atomic uses a non-standard FlexVolume plugin directory:  `/var/lib/kubelet/volumeplugins`
+The kubelet in OpenStack Magnum is already configured to use that directory.
+You will need to use this value when [Configuring the Rook operator](#configuring-the-rook-operator)
```

Comment:
Only a small thing here, the first char of `Configuring` should be lower case :wink:

Reply:
Whoops!